src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultTag/styles.ts

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+import Icon from '../../../ignitus-Utilities/Components/icon';
+import { StyledTagProps } from './types';
+
+export const StyledIcon = styled(Icon)`
+  height: 1.2rem;
+  width: 1.2rem;
+  fill: ${props => props.color};
+  transition: 0.3s;
+  flex: 1;
+  margin: 0.03rem;
+`;
+
+export const StyledTag = styled.span<StyledTagProps>`
+  background: ${props => props.background};
+  color: ${props => props.color};
+  font-weight: 600;
```

Comment:
Import the font as Paragraph and remove font-weight, font-size properties. So it will use that font-size, which will look smaller.

Reply:
![Screenshot from 2020-04-18 00-54-17](https://user-images.githubusercontent.com/43489708/79606401-2b598100-810f-11ea-87e1-d5a741d784ff.png)

This is after removing the `font-weight` and `font-size` properties. Pls see if it looks fine now.

Reply:
Yes, looks fine. Change the orange tag font-color to blue and for grey tag, white font-color should be there.

Reply:
Done